1905-09-14.xml

```diff
@@ -4644,32 +4554,11 @@
                         cultivable at irregular intervals, the fact that a person has cultivated the
                         same for whatever period shall not by itself give him the absolute ownership
-                        of the land. (iv.) The exercise by any person of rights in or over one or
-                        more portions or panels of tad shall not be taken as a presumption In his
-                        and nothing is at present known which should make the yield smaller than the
-                        last Europe during the past season has bought far less Surat than daring the
-                        preoeding season, but the Indian and Japanese mills have been busy. Tbe
-                        Indian holders of ootton have followed the Amerioan policy of refusing low
-                        prioes and have oarried over an unusually heavy stock into the new season.
-                        It remains to be seen bow far suoh a policy will answer their purpose. From
-                        Egypt the reports are distinctly good. Thera is saffioitnt water for the
-                        cultivator, although the volume of water fa the Nile has been only
-                        moderately full. A large acreage has been planted, and prospeots are at
-                        present flattering. The crop bos yet to fear autumn fogs, but if it escape
-                        these, or unless they are unusually injurious, a lull or large crop is to be
-                        looked for. It mould therefore appear that at the present moment the
-                        prospeots of all the principal crops an of a nature to encourage hopes of
-                        oootinned abundanoe. Even if the American orop should prove disappointing
-                        the large surplus earned over from the late orop will avert any real danger
-                        of shortage. Crops and surpluses to¬gether ought to supply every requirement
-                        of the spindles of the world for 1905-6. Appear¬ances, therefore, are in
-                        favor of another year’s healthy trade, and it can only be hoped that as the
-                        season pisses it may prove as good and as profitable for Lancashire as the
-                        season 1904-5. in Bogtieh will be resumed fa the Church of the American
-                        Mission Cairo, on the evening of the 17th of theonrrent month. The hour of
-                        service is from 6 p.m. to 7. Residents and visitors are cordially invited to
-                        attend. The Church is a part of the Missiom pre¬mises, boated almost fa
-                        front of Shepheard's Hotel. 26422-8-1 </p>
+                        of the land. (iv.)The exercise by any person of rights in or over one or
+                        more portions or panels of tad shall not be taken as a presumption In his </p>
+                </div>
```

Comment:
You can remove the div break around the column break. It can just say `his <cb n="2"/> favor`. Add paragraph breaks where appropriate.